.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "next/core-web-vitals"
```

Comment:
`core-web-vitals` tries to make us use the Script component instead of the browser supported synchronous script tag.